be/src/runtime/hdfs/hdfs_fs_cache.cpp

```diff
@@ -21,6 +21,23 @@
 
 namespace starrocks {
 
+// Try to get azure cloud properties from FSOptions, if azure credentials not existed, return nullptr.
+// TODO(SmithCruise): Should remove when using azure cpp sdk
```

Comment:
we use hdfs sdk to access azure in Fe and Be?

Reply:
Yes, more accurately, hdfs sdk in FE, LibHdfs in BE